ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/InterestMapper.java

```diff
@@ -37,8 +39,10 @@ public void to(Interest interest, CCDCase.CCDCaseBuilder builder) {
         interestBreakdownMapper.to(interest.getInterestBreakdown(), builder);
         interestDateMapper.to(interest.getInterestDate(), builder);
 
+        Optional.ofNullable(interest.getType())
```

Comment:
As per `uk.gov.hmcts.cmc.domain.models.Interest` this should not be null. 

```
@NotNull
    private final InterestType type;
```
Not sure why we still need to do this?

Reply:
I agree on it, but looks like claim store have some data